src/bundle/Resources/public/scss/fieldType/_ezrichtext.scss

```diff
@@ -10,6 +10,15 @@
         min-height: 100px;
         border: 1px solid rgba(0,0,0,0.15);
         border-radius: .25rem;
+        display: inline-block;
+        width: 100%;
+        padding: .5rem;
+
+        .is-block-focused,
+        .cke_widget_wrapper.cke_widget_focused>.cke_widget_element {
```

Comment:
Why so much specificity?
`.cke_widget_wrapper.cke_widget_focused > .cke_widget_element {`

Reply:
This one is from the CKEditor and I cannot replace it with something simpler.

Reply:
Ok, so please format the selector as I suggested above.